src/cloudai/workloads/moe_benchmark/throughput_reporter.py

```
@@ -0,0 +1,242 @@
+        f'<text x="{ml}" y="54" font-size="11" fill="#555">{html.escape(y_axis_label)}</text>',
+        f'<line x1="{ml}" y1="{y0}" x2="{ml + iw}" y2="{y0}" stroke="#333" stroke-width="1.5"/>',
+        f'<line x1="{ml}" y1="{mt}" x2="{ml}" y2="{y0}" stroke="#333" stroke-width="1.5"/>',
+    ]
+
+    for g in (0.25, 0.5, 0.75):
+        gy = y0 - g * ih
+        parts.append(f'<line x1="{ml}" y1="{gy:.1f}" x2="{ml + iw}" y2="{gy:.1f}" stroke="#ddd" stroke-width="1"/>')
+        gv = vmin + g * (vmax - vmin)
+        parts.append(f'<text x="{ml - 8}" y="{gy + 4:.1f}" font-size="11" text-anchor="end" fill="#444">{gv:.1f}</text>')
+
+    for cx, val, col, lab in zip(centers, values, colors, labels, strict=True):
```

Ruff (0.15.15)

[warning] Loop control variable `lab` not used within loop body

(B007)